tests/RazorLight.Tests/RazorLight.Tests.csproj

```diff
@@ -5,7 +5,15 @@
     <IsPackable>false</IsPackable>
     <PreserveCompilationContext>true</PreserveCompilationContext>
     <DefineConstants>$(DefineConstants);SOME_TEST_DEFINE</DefineConstants>
+  </PropertyGroup>
+  
+  <PropertyGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+    <LangVersion>7.3</LangVersion>
+  </PropertyGroup>
+  
+  <PropertyGroup Condition="'$(TargetFramework)' == 'netcoreapp3.0'">
     <LangVersion>8.0</LangVersion>
+    <IsTestProject>true</IsTestProject>
```

Comment:
Why is this only on netcoreapp3.0?

Reply:
In reading up on this. IsTestProject is set when your test project has a PackageReference to Microsoft.NET.Test.Sdk

Reply:
🤔 don’t remember adding it

Reply:
Let's remove it.